core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala

```diff
@@ -424,6 +428,22 @@ class BaseTiSparkTest extends QueryTest with SharedSQLContext {
     df.collect.foreach(println)
   }
 
+  def simpleSelect(dbName: String, dataType: String): Unit = {
```

Comment:
two `simpleSelect` can merge into one, INMO.

Reply:
It can be done, but not in this PR I think.